tests/baselines/reference/callbackTagNestedParameter.js

```diff
@@ -0,0 +1,92 @@
+//// [cb_nested.js]
+/**
+ * @callback WorksWithPeopleCallback
+ * @param {Object} person
+ * @param {String} person.name
+ * @param {Number} [person.age]
+ * @returns {void}
+ */
+
+/**
+ * For each person, calls your callback.
+ * @param {WorksWithPeopleCallback} callback
+ * @returns {void}
+ */
+function eachPerson(callback) {
+    callback({ name: "Empty" });
+}
+
+
+
+
+//// [cb_nested.d.ts]
+/**
+ * @callback WorksWithPeopleCallback
+ * @param {Object} person
+ * @param {String} person.name
+ * @param {Number} [person.age]
+ * @returns {void}
+ */
+/**
+ * For each person, calls your callback.
+ * @param {WorksWithPeopleCallback} callback
+ * @returns {void}
+ */
+declare function eachPerson(callback: WorksWithPeopleCallback): void;
+type WorksWithPeopleCallback = (person: 
+ * @param {String} person.name
+ * @param {Number} [person.age]
+) => void;

```

Comment:
I just generated these baselines for you, and unfortunately it looks like the declaration emitter is going to need some work in order to understand this. You can see here it’s emitted totally invalid syntax, pulling literal content from the middle of the JSDoc comment 😕

Reply:
Wow, this is new. I am sure this is a new problem, as this didn't happen when I tested this. As soon as I resolve the CLA issues, I might have a second look at this.

Reply:
I think you weren’t seeing this because you hadn’t yet generated declaration files via `@declaration: true`.

Reply:
Do you have any idea, why this use case is being handled differently by the emitter?

Reply:
I’m not sure without debugging.